packages/core-data/src/resolvers.js

```diff
@@ -409,15 +409,13 @@ export const getAutosave =
 export const __experimentalGetTemplateForLink =
 	( link ) =>
 	async ( { dispatch, resolveSelect } ) => {
-		// Ideally this should be using an apiFetch call
-		// We could potentially do so by adding a "filter" to the `wp_template` end point.
-		// Also it seems the returned object is not a regular REST API post type.
 		let template;
 		try {
-			template = await window
```

Comment:
I don't recall exactly why initially apiFetch didn't work for me 🤔 I wonder what changed.